docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1452,6 +1452,22 @@ As the core components use emotion as a styled engine, the props used by emotion
 
 ### withTheme
 
+- The `withTheme` HOC utility has been removed from the `@material-ui/core/styles` package. You can use `@material-ui/styles/withTheme` instead. Make sure to add a `ThemeProvider` at the root of your application, as the `defaultTheme` is no longer available. If you are using this utility together with `@material-ui/core`, it's recommended you use the `ThemeProvider` component from `@material-ui/core/styles` instead.
+
+  ```diff
+  - import { withTheme } from '@material-ui/core/styles';
```

Comment:
It renders funny:

```suggestion
  -import { withTheme } from '@material-ui/core/styles';
```

<img width="163" alt="Screenshot 2021-04-29 at 20 23 38" src="https://user-images.githubusercontent.com/3165635/116599780-d149e900-a928-11eb-9aac-0c2875579b4d.png">

Reply:
Updated 👍